nearpy/tests/experiments_tests.py

```diff
@@ -199,5 +199,69 @@ def test_random_binary_projections(self):
         print('\nRecall RBP: %f, Precision RBP: %f, SearchTime RBP: %f\n' % \
             (recall1, precision1, searchtime1))
 
+    def test_delete_vector(self):
```

Comment:
Why is it in this file? I suggest placing it in the `storage_tests` or `engine_tests`.